.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "plugins": ["es"],
+  "extends": [
+    "eslint:recommended"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
```

Comment:
And this?

Reply:
globals aren't needed - removing
plugins has the extra es6 specific rules
extends - the basic linting rules that eslint uses as standard
parserOptions - was in the base example

Reply:
ecmaVersion removed